src/main/java/io/codelex/dateandtime/practice/Exercise2/Exercise2.java

```diff
@@ -0,0 +1,41 @@
+package io.codelex.dateandtime.practice.Exercise2;
+
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class Exercise2 {
+    public static void main(String[] args) {
+        System.out.println("Please enter information below to specify server launch date.");
+        LocalDate date1 = userEnters(true);
+        System.out.println();
+        System.out.println("Please enter year (2022) and month (1-12) to check when you need to update server: ");
+        LocalDate date2 = userEnters(false);
+
+        System.out.println();
+        System.out.println("Server launch date: " + date1);
+        System.out.println();
+        
+
+        for (LocalDate date = date1; date.isBefore(date2); date = date.plusDays(14)) {
+            if (date.getMonth() == date2.getMonth() && date.getYear() == date2.getYear()) {
```

Comment:
arī palīgmetodi
in general pieturēties ne vaiŗak par 1-2 max operātori vienā rindā

Reply:
Hey, salabots.